JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -8,53 +8,43 @@ import Foundation
 
 // 과일 저장소 타입
 class FruitStore {
-    private var fruitInventory: [String: Int] = [
-        "딸기": 10,
-        "바나나": 10,
-        "파인애플": 10,
-        "키위": 10,
-        "망고": 10
-    ]
     
-    private func displayCurrentFruitInventory(fruit: String, quantity: Int) {
+    var fruitInventory: [String: Int]
+    
+    weak var delegate: QuantityViewControllerDelegate?
+    
+    init (fruitInit: [String : Int] = ["딸기": 10,"바나나": 10,"파인애플": 10,"키위": 10,"망고": 10 ]) {
```

Comment:
수정해보셨군요!👍
```suggestion
    init (fruitInit: [String : Int] = ["딸기": 10,"바나나": 10,"파인애플": 10,"키위": 10,"망고": 10]) {
```
마지막에 띄워쓰기가 들어가있네요 ㅎㅎ 사소한 부분도 챙겨주시면 좋을 것 같아요 ㅎㅎ